cite_seq_count/preprocessing.py

```diff
@@ -74,7 +74,7 @@ def parse_barcode_reference(
 
     else:
         barcodes_pl = barcodes_pl.with_columns(
-            reference=pl.col(REFERENCE_COLUMN).str.strip_chars(STRIP_CHARS),
+            reference=pl.col(required_header).str.strip_chars(STRIP_CHARS),
```

Comment:
in `barcodes_pl` there is not the 'reference' column (at least in the whitelist barcode)

Reply:
I'm changing the behavior here a little bit. I think I'm gonna use "reference" even in the whitelist input. Just makes the logic a bit cleaner